spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -104,7 +107,8 @@ public void setCredentialsProvider(CredentialsProvider credentialsProvider) {
 	}
 
 	/**
-	 * Set the provider for the HTTP headers to be added to the subscribers' REST API calls.
```

Comment:
Why all of the comment reformatting? It makes the PR harder to read. Can we leave the old formatting?